vignettes/pkgnet-intro.Rmd

```diff
@@ -0,0 +1,354 @@
+---
+title: "Exploring The Structure and Dependencies of An R Package"
+output: 
+  rmarkdown::html_vignette:
+    toc: true
+vignette: >
+  %\VignetteIndexEntry{Vignette Title}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setupVignette, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = FALSE,
+  comment = "#>",
+  fig.align = 'center'
+)
+```
+
+
+<!--  STYLE SET UP FOR TWO COLUMN SECTIONS  -->
+<style>
+.column-left{
+  float: left;
+  width: 50%;
+  text-align: left;
+}
+.column-right{
+  float: right;
+  width: 50%;
+  text-align: left;
+}
+</style>
+
+
+<!--  CODE BELOW HERE WILL RENDER  -->
+
+`pkgnet` is an R library designed for the analysis of R libraries! The goal of the package is to build a graph representation of a package and its dependencies to inform a variety of activities, including:
+
+- prioritizing functions to unit test based on their centrality
+- examining the recursive dependencies you are taking on by using a given package
+- exploring the structure of a new package provided by a coworker or downloaded from the internet
+
+Below is a brief tour of `pkgnet` and its features. 
+
+***
+# Running pkgnet
+
+`pkgnet` can analyze any R package locally installed.  Run `installed.packages()` to see the full list of packages installed on your system.  For this example, let's say we are analyzing a custom built package, `baseballstats`.
+
+```{r makeDemoPackage, include=FALSE}
+devtools::install_local(system.file('baseballstats',package="pkgnet"),force=TRUE)
+```
```

Comment:
I think this section should be shown (`include=TRUE`) rather than hidden. You could maybe title it "Follow along" and let people know that if they want to reproduce this example themselves, they can find `baseballstats` inside the `pkgnet` package and this is how they can install it.